Process.c

```diff
@@ -108,14 +113,43 @@ void Process_humanNumber(RichString* str, unsigned long long number, bool colori
       number %= 1000;
       len = xSnprintf(buffer, sizeof(buffer), "%03lluG ", number);
       RichString_appendnAscii(str, processGigabytesColor, buffer, len);
+   } else if (number < 100000 * ONE_M) {
+      //2 digit TB, 1 digit GB
+      number /= 100 * ONE_M;
+      len = xSnprintf(buffer, sizeof(buffer), "%2llu", number/10);
+      RichString_appendnAscii(str, largeNumberColor, buffer, len);
+      number %= 10;
+      len = xSnprintf(buffer, sizeof(buffer), ".%1llu", number);
+      RichString_appendnAscii(str, processGigabytesColor, buffer, len);
+      RichString_appendAscii(str, largeNumberColor, "T ");
+   } else if (number < 1000 * ONE_G) {
+      //3 digit TB
+      number /= ONE_G;
+      len = xSnprintf(buffer, sizeof(buffer), "%4lluT ", number);
+      RichString_appendnAscii(str, largeNumberColor, buffer, len);
+   } else if (number < 10000ULL * ONE_G) {
+      //1 digit PB, 3 digit TB
+      number /= ONE_G;
+      len = xSnprintf(buffer, sizeof(buffer), "%1llu", number/1000);
+      RichString_appendnAscii(str, largeNumberColor, buffer, len);
+      number %= 1000;
+      len = xSnprintf(buffer, sizeof(buffer), "%03lluP ", number);
+      RichString_appendnAscii(str, largeNumberColor, buffer, len);
```

Comment:
Why bother splitting this print when printing both parts the same color regardless?

Reply:
thanks, simplified